### ISSUE TITLE
www.orienttechnolgies.com is gone, downloading is completely broken now

### ISSUE THREAD

**Issue by athompso** [migibert/orientdb-role#30]
Getting this error:
```
TASK [migibert.orientdb : Download OrientDB tarball] ***********************************************************************************************************
task path: /root/.ansible/roles/migibert.orientdb/tasks/install.yml:1
fatal: [mysql03.vpc01.ibmcloud.onetrace.io]: FAILED! => {"changed": false, "dest": "/tmp/orientdb-3.0.23.tar.gz", "elapsed": 5, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-3.0.23.tar.gz&os=linux"}
fatal: [mysql02.vpc01.ibmcloud.onetrace.io]: FAILED! => {"changed": false, "dest": "/tmp/orientdb-3.0.23.tar.gz", "elapsed": 5, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-3.0.23.tar.gz&os=linux"}
fatal: [mysql01.vpc01.ibmcloud.onetrace.io]: FAILED! => {"changed": false, "dest": "/tmp/orientdb-3.0.23.tar.gz", "elapsed": 5, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-3.0.23.tar.gz&os=linux"}
```

which appears to directly caused by Orient Technologies not running that old URL anymore.

Ugh, download URLs are ugly... 

Version range|URL Format|Sample
---------------|-------------|--------
up to 2.2.19|download.php, but filenames change!|http://orientdb.com/download.php?email=unknown@unknown.com&file=orientdb-graphed-1.3.0.tar.gz&os=multi
2.2.20 - 2.2.22|bit.ly randomized url, redirects to download.php|https://bit.ly/2r1YBMi
2.2.23 - 2.2.34|bit.ly managed url, redirects to download.php|https://bit.ly/orientdb-ce-2-2-23
2.2.35+|S3 bucket|https://s3.us-east-2.amazonaws.com/orientdb3/releases/2.2.35/orientdb-community-2.2.35.tar.gz

Note: the bit.ly URLs for are the same for both 2.2.20 and 2.2.21.  \*sigh\*  I didn't bother checking to see which one got downloaded.  Presumably it can be bypassed by using the download.php link directly.

**Comment by athompso** [comment]
I see this is fixed in the github head (although downloading for older versions will now break instead, I think that's an acceptable trade-off) but the version installed through the "ansible-galaxy" command is old enough to still have the www.orienttechnologies.com address.
I know how to do a git clone and manually update the installed role file-by-file, but surely there's a better way?

**Comment by athompso** [comment]
Ah, I see the problem - Ansible Galaxy has v1.4 which was committed 3 years ago.
I figured out how to remove that version and install directly from github.
May I suggest tagging a v1.5 and uploading it so other even-more-clueless users aren't bitten by this?

**Comment by migibert** [comment]
Sure, I wanted to support download link for both old and new versions but I will release the v1.5 right now to let people install the new version. Thanks for reporting this!

**Comment by migibert** [comment]
In fact there is no need to support both versions as people can set the variable `orientdb_url` to use the old urls so I released 1.5.0 as is.

**Comment by athompso** [comment]
I had not noticed that variable.  That works, thanks!
